src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Refactoring/RenameEndpointTest.cs

```diff
@@ -628,17 +639,6 @@ private RenameEndpoint CreateEndpoint(LanguageServerFeatureOptions languageServe
 
         var projectSnapshotManagerDispatcher = new LSPProjectSnapshotManagerDispatcher(LoggerFactory);
 
-        _documentContextFactory = Mock.Of<DocumentContextFactory>(d =>
```

Comment:
This is not an important point to mention, I just wanted to express my frustration at the fact that this test defined `_documentContextFactory` twice, because my jet-lagged brain could not work out why my changes to the mock, a few lines above this, didn't fix the tests.